lib/galaxy/model/migrate/versions/0151_add_worker_process.py

```diff
@@ -0,0 +1,53 @@
+"""
+Add table for worker processes
+"""
+from __future__ import print_function
+
+import logging
+
+from sqlalchemy import (
+    Column,
+    DateTime,
+    MetaData,
+    Table,
+    TEXT,
+)
+
+from galaxy.model.orm.now import now
+
+log = logging.getLogger(__name__)
+metadata = MetaData()
+
+
+WorkerProcess_table = Table(
+    'worker_process',
+    metadata,
+    Column('server_name', TEXT, primary_key=True),
+    Column("update_time", DateTime, default=now, onupdate=now),
+)
+
+
+def upgrade(migrate_engine):
+    metadata.bind = migrate_engine
+    print(__doc__)
+    metadata.reflect()
+    _create(WorkerProcess_table)
+
+
+def downgrade(migrate_engine):
+    metadata.bind = migrate_engine
+    _drop(WorkerProcess_table)
+
+
+def _create(table):
```

Comment:
FYI, I'm refactoring these methods in https://github.com/galaxyproject/galaxy/pull/7621 to avoid the code duplications.